### PR TITLE
docs: specify custom arguments must have string values

### LINKED ISSUE #658: Document that custom_args properties must be strings (until fixed at API level)
#### Issue Summary

Fixes #288

### PR DESCRIPTION
**Pull request by OrthoDex**
# Fixes #658 

### Checklist
- [x] I have made a material change to the repo (functionality, testing, spelling, grammar)
- [x] I have read the [Contribution Guide] and my PR follows them.
- [x] I updated my branch with the master branch.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation about the functionality in the appropriate .md file
- [ ] I have added in line documentation to the code I modified

### Short description of what this PR does:
- Update mentions of `customArgs` field in the docs to reflect that it takes only string value
- Add an error handling section to the kitchen sink example to notify future users.

If you have questions, please send an email to [Sendgrid](mailto:dx@sendgrid.com), or file a Github Issue in this repository.

cc @thinkingserious

### CODE REVIEW

**Comment by SendGridDX** [comment]
[CLA assistant check](https://cla.sendgrid.com/sendgrid/sendgrid-nodejs?pullRequest=784) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.sendgrid.com/sendgrid/sendgrid-nodejs?pullRequest=784) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.sendgrid.com/check/sendgrid/sendgrid-nodejs?pullRequest=784) it.</sub>

File: use-cases/kitchen-sink.md [hunk 1]
```diff
@@ -48,3 +48,7 @@ sgMail
   .then(() => console.log('Mail sent successfully'))
   .catch(error => console.error(error.toString()));
 ```
+
+### Error Handling:
```
Comment (reedsa) [inline use-cases/kitchen-sink.md]: Not a big deal but perhaps this title should be `Caveats`?

Comment (OrthoDex) [reply use-cases/kitchen-sink.md]: agreed @reedsa , I'll update the PR

**Comment by OrthoDex** [comment]
Hi @reedsa, can this be merged in?

**Pull request merged**